From 6bcf96fca22f49e9d9382576e267ffc800cd39e6 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sat, 16 Dec 2023 09:27:23 -0800
Subject: [PATCH] VVI-flatbuffers-patch : Add module.bazel for bzlmod on
 v23.5.26

Summary:
Adds MODULE.bazel for bzlmod compatibility

Test Plan:
integration in vvplatform
---
 MODULE.bazel | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 MODULE.bazel

diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 00000000..12578b52
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,7 @@
+module(name = "flatbuffers", version = "23.5.26")
+
+bazel_dep(name = "aspect_rules_js", version = "1.34.1")
+bazel_dep(name = "protobuf", version = "21.7")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_nodejs", version = "6.0.2")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc1")
\ No newline at end of file
-- 
2.34.1

